dace/frontend/python/replacements.py

```diff
@@ -997,7 +997,20 @@ def _unop(sdfg: SDFG, state: SDFGState, op1: str, opcode: str, opname: str):
     return name
 
 
-def _broadcast_together(arr1_shape, arr2_shape):
+def _broadcast_to(target_shape, operand_shape):
```

Comment:
It would be nice if we can clarify what exactly is being supported here. For ufuncs, I made on purpose a new method to handle broadcast in order to understand how this works without potentially breaking expected/existing functionality.  I found that there are issues regarding symbols, e.g., which is greater; M or N. When it comes to symbolic sizes, what does `_broadcast_together` exactly support?. It would help with merging and removing duplicate functionality.